### PR TITLE
CDL: added code-group for more options

### PR DESCRIPTION
**Pull request by renejeglinsky**
@BraunMatthias and @agoerler could you check and add to the Java variants? Thank you :)

### CODE REVIEW

**Review by agoerler — COMMENTED** [review]
In CAP Java, during the Maven build doc comments are enabled automatically. This is important as the doc comments are propagated to the generated Java interfaces.(@mofterdinger - who is this achieved?)

However, in the runtime `CdsModel` doc comments are not present. Also the MTXS sidecar build creates CSN without doc comments.

File: cds/cdl.md [hunk 1]
```diff
@@ -1972,14 +1972,28 @@ CREATE TABLE Employees (
...
+{
+  "cds" : {
+    "docs" : true // [!code focus]
+  }
+}
+```
+```yaml [application.yaml]
+cds:
+  docs: true
+```
```
[the remark this answers is not in the document]
Comment (mofterdinger) [reply cds/cdl.md:1989]: Hi, the cds-maven-plugin enables the document generation via this env. variable `CDS_CDSC_DOCS = true`: https://github.wdf.sap.corp/cds-java/cds-services/blob/cc847b3eb9d87ff426a6c70f71a869aeab834f59/cds-maven-plugin/src/main/java/com/sap/cds/maven/plugin/build/CdsMojo.java#L171

[the remark this answers is not in the document]
Comment (renejeglinsky) [reply cds/cdl.md:1989]: So the `.cdsrc.json` configuration is needed then?

[the remark this answers is not in the document]
Comment (renejeglinsky) [reply cds/cdl.md:1989]: @mofterdinger Could you add the needed `.cdsrc.json` config? Would be sth like this, right?
```json [.cdsrc.json]
	{"cdsc" : {
	  "docs" : true // [!code focus]
	}}
```

[the remark this answers is not in the document]
Comment (agoerler) [reply cds/cdl.md:1989]: No config is needed for CAP Java. Doc comments are enabled automatically by the Maven build.

File: cds/cdl.md [hunk 2]
```diff
@@ -1982,17 +1982,32 @@ CREATE TABLE Employees (
...
```
Comment (renejeglinsky) [inline cds/cdl.md:2005]: As I understood, there's no config needed for Java:
```suggestion

```

File: cds/cdl.md [hunk 3]
```diff
@@ -1982,17 +1982,28 @@ CREATE TABLE Employees (
...
+    "docs" : true // [!code focus]
+  }
+}
+```
+```js [JavaScript]
 cds.compile(..., { docs: true })
 ```
+:::
 
 ::: tip
```
Comment (renejeglinsky) [inline cds/cdl.md:2007]: CAP Java: Doc comments enabled by default

Comment (renejeglinsky) [inline cds/cdl.md:2007]:
```suggestion
:::

::: tip CAP Java: Doc comments enabled by default
```

Comment (agoerler) [reply cds/cdl.md:2007]: Or

```suggestion
:::

::: tip Doc comments are enabled by default in CAP Java 😀.
```

**Review by agoerler — APPROVED** [review]

**Pull request merged**